src/cgnslib.h

```diff
@@ -455,10 +455,11 @@ typedef enum {
   CGNS_ENUMV( IFaceCenter ) =5,
   CGNS_ENUMV( JFaceCenter ) =6,
   CGNS_ENUMV( KFaceCenter ) =7,
-  CGNS_ENUMV( EdgeCenter ) =8
+  CGNS_ENUMV( EdgeCenter ) =8,
+  CGNS_ENUMV( ElementBased ) =9
```

Comment:
ElementBased --> InterpolationPoints

as ElementBased is too generic... and confusing.
ElementInterpolationPoints would be great but maybe too long

Reply:
good point, fixed.